build_tools/azure/py38_conda_defaults_openblas_environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - numpy=1.17.3  # min
   - blas[build=openblas]
   - scipy=1.5.0  # min
-  - cython<3.0.0
+  - cython=0.29.33  # min
```

Comment:
Since other dependencies used min dependencies on this build I decided to follow this rule for cython as well.